CHANGELOG.rst

```diff
@@ -15,21 +30,21 @@ Preparation for the promotion of the ``aws_ssm.py`` plugin (https://forum.ansibl
 Minor Changes
 -------------
 
-- aws_ssm - Refactor ``_init_clients`` method for improved clarity and efficiency (https://github.com/ansible-collections/community.aws/pull/2223).
-- aws_ssm - Refactor ``_prepare_terminal()`` method for improved clarity and efficiency (https://github.com/ansible-collections/community.aws/pull/).
-- aws_ssm - Refactor ``exec_command`` method for improved clarity and efficiency (https://github.com/ansible-collections/community.aws/pull/2224).
-- aws_ssm - Add the possibility to define ``aws_ssm_plugin`` variable via environment variable and by default use the version found on the ``$PATH`` rather than require that you provide an absolute path (https://github.com/ansible-collections/community.aws/issues/1990).
-- aws_ssm - Add function to generate random strings for SSM CLI delimitation (https://github.com/ansible-collections/community.aws/pull/2235).
-- dms_endpoint - Improve resilience of parameter comparison (https://github.com/ansible-collections/community.aws/pull/2221).
+- aws_ssm -  Refactor ``_init_clients`` Method for Improved Clarity and Efficiency (https://github.com/ansible-collections/community.aws/pull/2223).
+- aws_ssm -  Refactor ``_prepare_terminal()`` Method for Improved Clarity and Efficiency (https://github.com/ansible-collections/community.aws/pull/).
+- aws_ssm -  Refactor exec_command Method for Improved Clarity and Efficiency (https://github.com/ansible-collections/community.aws/pull/2224).
+- aws_ssm - Add the possibility to define ``aws_ssm plugin`` variable via environment variable and by default use the version found on the $PATH rather than require that you provide an absolute path (https://github.com/ansible-collections/community.aws/issues/1990).
+- dms_endpoint - improve resilience of parameter comparison (https://github.com/ansible-collections/community.aws/pull/2221).
```

Comment:
I don’t quite understand these changes. The changelog formats seem inconsistent. Should we enforce a standard format? If so, what should it be?

Reply:
What's happening here is that during the previous release formatting fixes were applied to the RST rather than to changelog.yaml and then merged.  Because the RST is automatically generated from changelog.yaml, now that the RST is being **re**generated from the YAML, those manual changes are being stripped out again.

In part this is a training/process issue.  However, we could potentially fix this with a technical change, that automatically generates the RST from the YAML and adds an extra commit to the PR (see also the [update variables workflow](https://github.com/ansible-collections/community.aws/blob/main/.github/workflows/update-variables.yml)